lib/feed_item_accessor.rb

```diff
@@ -12,6 +12,12 @@ def element_content(element_name)
   end
 
   def link
+    if rss_item.respond_to?(:links) && links = rss_item.links
```

Comment:
I think using assignment in conditionals is fine but only when it's the only condition. When it's mixed with other conditions with && or ||, then it becomes quite confusing 🤔

Reply:
Do you prefer another nested if? I do not like it either, but I hate many nested ifs a little more.

Reply:
Something like this?

```ruby
def link
  if rss_item.respond_to?(:links)
    if link = rss_item.links&.find { |l| l.rel == "alternate" && l.type == "text/html" }
      return link.href if link.respond_to?(:href)
    end
  end

  try_attribute_or_self(element(:link), :href)
end
```